packages/kbn-dom-drag-drop/src/sass/drag_drop.scss

```diff
@@ -15,7 +15,7 @@
   width: 100%;
   left: 0;
   opacity: .9;
-  transform: translate($euiSizeXS, $euiSizeXS);
+  transform: translate(-$euiSizeS, -$euiSizeL);
```

Comment:
Would it be fine to keep positive offsets? Otherwise in Discover it's cut off.

With `transform: translate($euiSizeXS, $euiSizeXS);`:
<img width="500" alt="Screenshot 2023-06-14 at 10 41 58" src="https://github.com/elastic/kibana/assets/1415710/0b9051b5-5f1e-48cd-a90b-f0813fab7b80">

With `transform: translate(-$euiSizeS, -$euiSizeL);`:
<img width="500" alt="Screenshot 2023-06-14 at 10 41 13" src="https://github.com/elastic/kibana/assets/1415710/fed6ec7e-9def-42ee-9243-e184b952be4a">

Reply:
Thank you so much for noticing! I think doing the opposite values will work fine both for Lens and Discover, WDYT?
<img width="289" alt="Screenshot 2023-06-14 at 13 54 11" src="https://github.com/elastic/kibana/assets/4283304/4ebfbc74-10e6-422f-a10f-f38f2108c487">
<img width="696" alt="Screenshot 2023-06-14 at 13 54 38" src="https://github.com/elastic/kibana/assets/4283304/6a1dba8d-4ce9-4f81-8090-e575de050c3d">

Reply:
Yes, looks great! Thanks!